src/test/java/com/puppycrawl/tools/checkstyle/api/FilterSetTest.java

```diff
@@ -45,48 +44,62 @@ public void testRemoveFilters() {
         final Filter filter = new SeverityMatchFilter();
         filterSet.addFilter(filter);
         filterSet.removeFilter(filter);
-        assertEquals(0, filterSet.getFilters().size(), "size is the same");
```

Comment:
can we use `Invalid filter set size` instead of `size is the same` please, since it makes more sense

Reply:
please fix

Reply:
done